code/modules/research/machinery/_production.dm

```diff
@@ -239,35 +265,47 @@
 /obj/machinery/rnd/production/proc/design_menu_entry(datum/design/D, coeff)
```

Comment:
This procs output is cached, right?

Reply:
It doesn't seem to be? It seems like it's just piped to the UI.

Reply:
ah right, that's why the rnd machines cause so much overtime. I forgot